grouper/fe/templates/macros/ui.html

```diff
@@ -87,11 +87,25 @@ <h3 class="panel-title">My Groups</h3>
     </div>
 {%- endmacro %}
 
-{% macro account(user) -%}
-<a href="/{{user.type|lower}}s/{{user.name}}">
-    <i class="fa fa-user{% if user.type == 'Group' %}s{% endif %}"></i> 
+{% macro account(user, type=None) -%}
+{% if type == None %}
+<a href="/{{ user.type | lower }}s/{{ user.name }}">
+    <i class="fa fa-user{% if user.type.lower() == 'group' %}s{% endif %}"></i> 
     {{ user.name }}
 </a>
+{% else %}
+<a href="/{{ type | lower }}s/{{ user }}">
+    <i class="fa fa-user{% if type.lower() == 'group' %}s{% endif %}"></i> 
+    {{ user }}
+</a>
+{% endif %}
+{%- endmacro %}
+
+{% macro permission(perm) -%}
+<a href="/permission/{{perm.name or perm.permission}}">
+    <i class="fa fa-key"></i>
+    {{perm.name or perm.permission}}
+</a>
 {%- endmacro %}
 
 {% macro permission(perm) -%}
```

Comment:
Pretty sure this is an amazing failure of a patch merge that ended up creating a duplicate section.